pkgs/development/python-modules/debugpy/default.nix

```diff
@@ -49,14 +59,23 @@ buildPythonPackage rec {
   )'';
 
   checkInputs = [
-    colorama django flask gevent psutil pytest
-    pytest-timeout pytest_xdist requests
+    colorama
+    flask
+    gevent
+    psutil
+    pytest
+    pytest-timeout
+    pytest_xdist
+    requests
+  ] ++ stdenv.lib.optional (!isPy27) [
+    django
   ];
 
   # Override default arguments in pytest.ini
   checkPhase = "pytest --timeout 0 -n $NIX_BUILD_CORES"
-               # gevent fails to import zope.interface with Python 2.7
-               + stdenv.lib.optionalString isPy27 " -k 'not test_gevent'";
+    # django 1.11 is no longer supported (last version to support Python 2.7) &
+    # gevent fails to import zope.interface with Python 2.7
+    + stdenv.lib.optionalString isPy27 " -k 'not test_django and not test_gevent'";
```

Comment:
```suggestion
    + lib.optionalString isPy27 " -k 'not test_django and not test_gevent'";
```

Reply:
This is also a scenario where the `pytestCheckHook` and `disabledTests` could come in handy.

Reply:
Yep, thank you. I already finished the changes since you mentioned them to me in the other code review. Just testing the builds now.